packages/flutter/test/material/scrollbar_paint_test.dart

```diff
@@ -45,7 +45,7 @@ void main() {
         )
         ..rect(
           rect: const Rect.fromLTRB(796.0, 1.5, 800.0, 91.5),
-          color: const Color(0x1a000000),
+          color: const Color(0xffbcbcbc),
```

Comment:
Defining a color constant whose name helps the reader understand where the color is defined would be better than repeating the same literal.